src/main/lrsql/util.clj

```diff
@@ -14,7 +14,8 @@
   (:import [java.util UUID]
            [java.time Instant]
            [java.io StringReader PushbackReader ByteArrayOutputStream]
-           [java.nio.charset Charset]))
+           [java.nio.charset Charset]
+           [java.util Calendar]))
```

Comment:
You can put `Calendar` alongside `UUID` since they're both being imported from `java.util`.

Reply:
nice catch, missed that entirely. https://github.com/yetanalytics/lrsql/pull/300/commits/c22c56150325dbde64bdd46941cb1cddae7cfeeb